src/atlas/field/FieldSet.cc

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright 2013 ECMWF.
+ * (C) Copyright 2013-2023 ECMWF.
```

Comment:
Not necessary to add "2023" (here and other file). We assume an open-ended approach.

Reply:
... done